prow/config.yaml

```diff
@@ -5376,6 +5376,20 @@ presubmits:
       - image: gcr.io/mlkube-testing/builder:latest
         imagePullPolicy: Always
 
+  tensorflow/minigo:
+  - name: tf-minigo-presubmit
+    context: tf-minigo-presubmit
+    agent: kubernetes
+    always_run: true         # Run for every PR, or only when requested.
+    branches:
+    - master
+    labels:
+      preset-service-account: true
```

Comment:
it might not need our service account? Where's the job upload logs to?

Reply:
Not sure if this is intended but this also doesn't seem to be doing git checkout etc. ?

Reply:
It doesn't upload the logs anywhere. But it seems like that would be a useful feature! What's the pattern here?

Reply:
I left a comment related to this [here](https://github.com/tensorflow/minigo/pull/40#discussion_r165541198).